src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.html

```diff
@@ -43,6 +43,23 @@
 
 <div class="edit-sqi" *ngIf="exercise.gradingInstructionFeedbackUsed">
     <div class="grading-instructions-update-border">
+        <div class="instructions" style="padding-bottom: 12px">
+            <jhi-markdown-editor
+                #markdownEditor
+                class="editor"
+                style="width: 65%"
+                [markdown]="markdownEditorText"
+                [domainCommands]="domainCommandsGradingInstructions"
+                (markdownChange)="prepareForSave()"
+                (textWithDomainCommandsFound)="setExerciseGradingInstructionText($event)"
+                [enableFileUpload]="false"
+                [shouldDisplayAlert]="false"
+                [enableResize]="true"
+                [showEditButton]="false"
+                [showPreviewButton]="false"
+            ></jhi-markdown-editor>
+            <div class="form-group text-center" style="width: 25%"></div>
```

Comment:
I am curious if we could use the bootstrap grip system in order to avoid width percentages, as described here: https://getbootstrap.com/docs/5.0/layout/grid/#row-columns
What do you think?

Reply:
hmm, coool. worth a try